### ISSUE TITLE
[WhatsApp][FFOS 2.2] [0.5.19 beta] cannot register via SMS or Call after new install

### ISSUE THREAD

**Issue by MartinSauer** [loqui/im#1238]
Response: {{SMSreason_undefined}}

**Comment by MartinSauer** [comment]
After getting the registration code via 0.5.17 the 0.5.19 is running (beta is gone). But I still can´t send messages to groups.

**Comment by bqphone** [comment]
I have the same problem as Martin.

Bq e4.5 phone just flashed it to the latest ubports ubuntu version, so i had to reinstall the apps again.

using now loqui IM 0.5.20, registered before using 0.5.17

**Comment by 03927e9c** [comment]
Yeah I have the same problem sending messges to private chats or groups. SMS worked fine.

**Comment by Zeus76** [comment]
I installed the version 0.5.19 beta on my Firefox OS 1.3, and did not send the registration message.

I got the "log" file and it says "old version".

Here's the file:
[2017-06-20_0 (1).txt](https://github.com/loqui/im/files/1087946/2017-06-20_0.1.txt)


Anyway he has sent me a message from the WebIDE simulator, with Firefox OS Simulator. But the message does not reach the recipient, it stays on the server.

I hope it helps you.

**Comment by bqphone** [comment]
upated to 05.21 i got: "couldn't send SMS! {{SMS_reason_undefined}}, i have tried another number and it worked. Is that because i had it working on this number before the app was removed due to flashing ubuntu touch to ubports version?

**Comment by 03927e9c** [comment]
05.21 is not released yet... maybe there are some bugs?

**Comment by bqphone** [comment]
well on ubuntu touch the version is 0.5.21 installed from openstore, if i click  and keep on pressing on LoquilM icon it open in ubuntu store and i see there version 0.5.17.

Basically it will register with sms if i use a new number, but for my current number which i have already registered a while ago, it will not accept it. So maybe this will help you fix the problem? need more info feel free to ask.
